iosHackthonApp/iosHackthonApp/SignUp/CommonView/SignUpInfoView.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
Do we think this should live in the Components module?

Reply:
Done..moved to Components module